From c22a5aaf45d21487641080d185e175d7d78d65d1 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Tue, 30 Aug 2011 07:55:17 +0000
Subject: [PATCH] script: fix building with --disable-nls

We should only include the libintl.h when NLS is requested.

This fixes issue when building util-linux with uClibc.

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 term-utils/script.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/term-utils/script.c b/term-utils/script.c
index d96ca2f..99748a1 100644
--- a/term-utils/script.c
+++ b/term-utils/script.c
@@ -55,7 +55,9 @@
 #include <getopt.h>
 #include <unistd.h>
 #include <fcntl.h>
+#ifdef ENABLE_NLS
 #include <libintl.h>
+#endif
 #include <limits.h>
 #include <locale.h>
 #include <stddef.h>
-- 
1.7.6.1

